src/main/resources/jenkins/branch/PropertyMigration/MonitorImpl/message.properties

```diff
@@ -0,0 +1,26 @@
+#
+# The MIT License
+#
+# Copyright (c) 2018, CloudBees, Inc., Stephen Connolly.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+blurb=Some of your jobs are using deprecated functionality that has been replaced by functionality in a new plugin. <br/>\
+  Please install the following plugin(s):<pre>{0}</pre>
```

Comment:
The admin monitor seems like overkill to me.

Suggest a simpler path. Add form validation for all this stuff saying that it is deprecated and to use the new plugin.  That plugin should convert existing folders in an initializer when it starts up, and use `DescriptorVisibilityFilter` to hide the old features. And we can add the new plugin to `platform-plugins.json`, `workflow-aggregator` dep, whatever, so that people will tend to get it without asking.